scm_v3c/radio.c

```diff
@@ -250,12 +250,20 @@ void repeat_rx_tx(repeat_rx_tx_params_t repeat_rx_tx_params) {
         cfg_mid_start = repeat_rx_tx_params.fixed_lc_mid;
         cfg_fine_start = repeat_rx_tx_params.fixed_lc_fine;
 
+        // 1.1V (NOP) VDDD tap fix
+        // the NOPs add some extra delay
+        // print statement also needed to be removed
+        // probably could be added back in if broken down into more
+        // than one print statement
         cfg_coarse_stop = cfg_coarse_start + 1;
+        __asm("NOP");
```

Comment:
Strange that we need a 1 cycle pause between writes, even though we aren't reading those values until later!